pkgs/by-name/br/broadcast-box/package.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  nixosTests,
+  fetchFromGitHub,
+  buildNpmPackage,
+  buildGoModule,
+}:
+let
+  name = "broadcast-box";
+  version = "2024-07-15";
```

As per https://github.com/NixOS/nixpkgs/blob/3540ebe5534a9e7f38b3c782dba8a03eaeae7303/pkgs/README.md#versioning this should start with `0-unstable-`, unless upstream starts tagging commits.

```suggestion
  version = "0-unstable-2024-07-15";
```